<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CVLabApp</class>
 <widget class="QMainWindow" name="CVLabApp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>691</width>
    <height>535</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CVLabApp</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="beforeLabel">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>10</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Image Processing</string>
    </property>
   </widget>
   <widget class="QLabel" name="srcImgLabel">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>50</y>
      <width>500</width>
      <height>250</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>500</width>
      <height>250</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>500</width>
      <height>250</height>
     </size>
    </property>
    <property name="text">
     <string>srcImg</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>40</y>
      <width>541</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>300</y>
      <width>531</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_5">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>50</y>
      <width>20</width>
      <height>251</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_6">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>50</y>
      <width>20</width>
      <height>251</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="openFileBtn">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>320</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>打开文件</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addNoiseBtn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>AddNoise</string>
    </property>
   </widget>
   <widget class="QPushButton" name="AMFBtn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>350</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>AMF</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>691</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
